src/tests/testIoManip.cc

```diff
@@ -153,6 +155,31 @@ TestIoManip::testAsHex()
     resetStream(ss);
 }
 
+void
+TestIoManip::testAtMostOnce()
+{
+    {
+        std::ostringstream ss;
+        auto textOnce = AtMostOnce("text1");
+        ss << textOnce;
+        CPPUNIT_ASSERT_EQUAL(std::string("text1"), ss.str());
+        ss << textOnce;
+        ss << textOnce;
+        CPPUNIT_ASSERT_EQUAL(std::string("text1"), ss.str());
+    }
+
+    {
+        std::ostringstream ss;
+        const std::string s("text2");
+        auto textOnce = AtMostOnce(s);
```

Comment:
If we want to document std::string lifetime considerations, we could add a comment like this:

```suggestion
        // Cannot create std::string when creating textOnce because the string may be
        // destroyed before we are done with textOnce:
        // auto textOnce = AtMostOnce(std::string("do not do this"));
        const std::string s("text2");
        auto textOnce = AtMostOnce(s);
```

I did not add it myself because it feels a bit too obvious and because few developers will notice/read this comment when working with AtMostOnce. I am not against adding it.

Reply:
It won't hurt, and I see value in having tests as a source of live documentation if anyone needs it, however trivial